.github/styles/Google/AMPM.yml

```diff
@@ -0,0 +1,9 @@
+extends: existence
```

Comment:
:memo: All of the files under `.github/styles` are downloaded when you run `vale sync` which takes configuration from `.vale.ini` file [here](https://github.com/keptn-sandbox/new-keptn-docs-engine/pull/20/files#diff-0da89932043a07a361f29194237fa0cb85aba0bebef98a012a03f84989c26e2e)

![image](https://user-images.githubusercontent.com/34534103/185552510-71f261ab-a08e-46b8-bc6f-468358f62be8.png)